.github/CONTRIBUTING.md

```diff
@@ -1,28 +1,38 @@
 # Contributing guidelines
 
-For anyone who knows JavaScript basics: feel free to contribute to this repository! Our code is open source. Be sure to follow our [code of conduct](https://github.com/ScratchAddons/ScratchAddons/blob/master/.github/CODE_OF_CONDUCT.md).
+For anyone who knows JavaScript basics: feel free to contribute to this repository! Our code is open source.
 
-## Filing [issues](https://docs.github.com/en/github/managing-your-work-on-github/about-issues)
+Be sure to follow our [code of conduct](https://github.com/ScratchAddons/ScratchAddons/blob/master/.github/CODE_OF_CONDUCT.md).
 
-Found a bug? Report it in an issue! But first, be sure to check if it was already reported by using the search bar [here](https://github.com/ScratchAddons/ScratchAddons/issues). If there are no similar issues, you can create a new one. We will take a look at it.
+## Reporting bugs and suggesting features
+
+Found a bug? You can open an [issue](https://github.com/ScratchAddons/ScratchAddons/issues) to report it! But first, be sure to check if it was already reported by using the search bar [here](https://github.com/ScratchAddons/ScratchAddons/issues). If there are no similar issues, you can create a new one. We will take a look at it.
 
 Also, feel free to ask questions, suggest new features, and more.
 
 Another way to suggest an idea or to ask a question is to start a [discussion](https://github.com/ScratchAddons/ScratchAddons/discussions).
 
-## Creating [pull requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+## Making contributions
+
+Contributions are welcome! If you know how to fix an issue, you can create a pull request to propose the necessary changes which we can then add to Scratch Addons.
+
+### Learn more about making addons
+
+If you're making a new addon, our ["Creating an Addon" FAQ page](https://scratchaddons.com/docs/develop/getting-started/creating-an-addon/) will teach you how to make one.
```

Comment:
Instead of "FAQ page", let's just do "documentation page" or "page".

Reply:
Oops!

Reply:
I would say a guide would be a little too much, but it is fine for me.